testsuite/smoke-test/src/full_nodes.rs

```diff
@@ -179,16 +182,8 @@ async fn test_vfn_failover() {
     )
     .await;
 
-    for _ in 0..8 {
-        transfer_coins_non_blocking(
-            &vfn_client,
-            &transaction_factory,
-            &mut account_0,
-            &account_1,
-            1,
-        )
-        .await;
-    }
+    assert_balance(&vfn_client, &account_0, 9).await;
```

Comment:
why is this assert instead of the transfer?

Reply:
These changes aren't too significant.

In this case, it seems more clear that we're checking that the transfer went through. (Even though transfer_coin also checks this, although not the balance).